### PR TITLE
add group language and remove grading info

### PR DESCRIPTION
**Pull request by gadamico**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,135 +136,6 @@ Ideally, your README should include:
 
 The best practice would be to include all of the items listed above in your README. Items 2 and 7 are particularly relevant for grading, as described below.
```
Comment (jessepisel) [inline README.md]: Does the last sentence need to be here since we are removing grading?

Comment (jessepisel) [reply README.md]: deliverables and getting started can both be combined into a single item

**Review by jessepisel — CHANGES_REQUESTED** [review]
A couple of minor requests

**Review by bpurdy-ds — APPROVED** [review]
Looks good.

**Pull request merged**